src/_guides/language/effective-dart/style.md

```diff
@@ -228,6 +228,24 @@ Db
 {% endprettify %}
 
 
+### DON’T use prefix letters to indicate type, mutability, or scope.
+
+The type system can tell you all about these for any identifier,
+so there's no need to use
+[Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)
+or any other prefix scheme to encode this information.
```

Comment:
It's not *quite* the type system that tracks things like scope and mutability. Maybe:

```
[Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) and other
schemes arose in the time of BCPL when the compiler didn't do much to help you
understand your code. In Dart, the language will happily tell you the type,
scope, mutability, and other properties of your declarations, so there is no
reason to encode it in the name.
```